src/components/NewsletterSignup/_newsletter-signup.scss

```diff
@@ -21,15 +21,17 @@
   z-index: var(--z-low);
 
   // WYSIWYG content styles
+  .cc-rich-text a,
```

Comment:
Needed to override the [rich text styles](https://github.com/wellcometrust/corporate-components/blob/develop/src/components/RichText/_rich-text.scss#L143)